ITE/6/README.md

```diff
@@ -0,0 +1,574 @@
+three distinct steps in verification. The innermost layer is user-defined to
+allow customers to define their own schemas; "link" is now one such user-defined
+schema.
+
+This specification is developed jointly with
+[Binary Authorization](https://cloud.google.com/binary-authorization), who will
+support the agreed-upon format once finalized. This way a single link file will
+be usable by either system.
+
+# Goals
+
+*   Standardize artifact metadata without being specific to the consumer (e.g.
+    in-toto or Binary Authorization). This way CI/CD pipelines, vulnerability
+    scanners, and other systems can generate a single set of attestations that
+    can be consumed by anyone.
+
+*   Make it possible to write policies (layouts) that take advantage of
+    structured information.
+
+# Specification
+
+## Introduction
+
+An **attestation** is the generalization of an in-toto link. It is a statement
+about an artifact, signed by an attester. Each attestation has a type indicating
+what the statement means, plus type-dependent details. In this new model, a
+"link" is just one type of attestation, but others are possible.
+
+Examples of attestations:
+
+*   Provenance: GitHub Actions attests to the fact that it built a container
+    image with digest "sha256:87f7fe…" from git commit "f0c93d…" in the "master"
+    branch of "https://github.com/example/foo".
+*   Code review: GitHub attests to the fact that Alice uploaded and Bob approved
+    git commit "f0c93d…" in the "master" branch of
+    "https://github.com/example/foo".
+*   Test result: Google Container Analysis attests to the fact that no
+    vulnerabilities were found in container image "sha256:87f7fe…" at a
+    particular time.
+*   Policy decision: Binary Authorization attests to the fact that container
+    image "sha256:87f7fe…" is allowed to run under GKE project "example-project"
+    within the next 4 hours, and that it used the three attestations above and
+    as well as the policy with sha256 hash "79e572".
+
+The benefit of this ITE is to express these attestations more natually than was
+possible with the old in-toto link schema.
+
+## Authentication and serialization
+
+The authentication and serialization is handled by
+[ITE-5](https://github.com/MarkLodato/ITE/blob/ite-5/ITE/5/README.md). The
+payload is always JSON encoded with a `payloadType` of
+`"https://in-toto.io/Attestation/v1-json"`. (In the future other encodings, such
+as CBOR, may be defined.)
+
+Prior to ITE-5, the existing in-toto signature wrapper may be used. In this
+case, the payload is always JSON with a `_type` of
+`"https://in-toto.io/Attestation/v1-json"`.
+
+## Schema
+
+*   [Attestation v1](spec/attestation.md) (base class)
+*   [Provenance v1](spec/provenance.md)
+*   [Link v1](spec/link.md) (matches [in-toto 0.9])
+
+We expect to a few more standard attestation types over time, and customers may
+define their own custom attestation types if desired.
```

Comment:
Is there a mechanism for registering types? Given they are represented by URIs / URLs, is there a meaning behind what these resolve to (if at all)?

Reply:
No. This is now stated explicitly.